src/kudu/consensus/consensus.proto

```diff
@@ -545,11 +545,16 @@ message RunLeaderElectionRequestPB {
 
   // Time when the original server was promoted away from. We can use this to
   // measure the total time of a chain of promotions
+  // Should be specified as nanoseconds since epoch
```

Comment:
now that I think of it, was it better to have a simple trace_id field with flexible schema?

Reply:
Do you mean json like field? Or an actual id with storage somewhere?

Was thinking that for the first, given that protobuf itself is supposed to help with protocol changes, and changing the json definition still needs changes on both sides, we probably don't gain too much?

Reply:
concern was that it has already cluttered up the RunLeaderElectionRequestPB with many fields.
This area is also evolving so in the future we might need more fields.

Reply:
My preference would be to have things schematised instead of having dynamic jsons just so protocol changes are less risky. What can be done here however is that we can collate this into another PB struct. What do you think about that?